main.py

```diff
@@ -0,0 +1,21 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+def get_recipe_price(prices, optionals=[], **ingredients):

```

Comment:
Add documentation please :)

Reply:
Remember to use typing hints and return types.

Reply:
Yes I did it right now.

Reply:
1. Fix the documentation.
2. You still didnt add typing hints :)